src/features/collapsible-sidebar/CollapsibleSidebar.js

```diff
@@ -92,11 +92,10 @@ class CollapsibleSidebar extends React.Component<Props> {
             'bdl-CollapsibleSidebar',
             className,
         );
-        const wrapperClasses = classNames('bdl-CollapsibleSidebar-wrapper', { hidden: isHidden });
```

Comment:
What's the reason for removing `isHidden`?

@jcamden Are the changes in this PR still needed? https://github.com/box/box-ui-elements/pull/2844
If not, we should probably remove the prop, `aria-hidden`, and the tests associated with `isHidden`.

Reply:
isHidden was being used before to show or not show the sidebar which led to issues because we need the sidebar to be controlled by css-only since it has transitions. 

@jcamden Do we need this aria-hidden attribute on the wrapper element?

Reply:
Hey @tjuanitas , thanks for commenting! To put a finer point on what Josh said above, unless I misunderstand, we cannot show or hide CollapsibleSidebar via css-only. Previously, we used JS to conditionally add a `hidden` class to the sidebar to set `display:none`. Josh's approach leverages the existing `isExpanded` class (which is, of course, conditionally added via JS) to set the width of the sidebar, fixed width by default, and `100vw` on mobile screen widths. The width of the sidebar is also set to `0vw` by default on mobile screen widths, so that it toggles between hidden and fullscreen, without the use of `width: 100%` or `display:none`, since both of those styles interfered with transitions.

The implications of this are interesting. It turns out that I didn't need to add `aria-hidden={true}` for `display:none`. Evidently, it is set by default. Funny enough, now that we are using `0vw`, I believe we _do_ need to manually set `aria-hidden={true}`. @tjuanitas , you are 100% correct that the we need to make changes to the tests that depend on `isHidden`. Thank you so much for catching that!

Reply:
Gotcha. So then the `isHidden` prop is now solely used for setting `aria-hidden`, correct? I think the tests may be fine at the moment but let's update the prop comment:
`/** Optionally apply hidden class (__visibility.scss) and "aria-hidden": "true" to CollapsibleSidebar wrapper */`

Reply:
@tjuanitas We don't use the visibility.scss file here. The sidebar is shown with a transition via its width which is either 0 or "full screen". 

In that case, the comment should be updated to 
`/** Optionally apply "aria-hidden": "true" to CollapsibleSidebar wrapper */`

Reply:
Yeah sorry I meant let's update [this line](https://github.com/box/box-ui-elements/blob/f722e7d59a7522ade6ec2bb0e80a8847bffd3028/src/features/collapsible-sidebar/CollapsibleSidebar.js#L40) to reflect the new usage of the prop. The wording you used is good with me.

Reply:
Out of curiosity, is there a way we could set `aria-hidden` automatically (and reliably) based on detecting the screen size and using the `expanded` prop? That way we don't need to use `isHidden` since I imagine a lot of developers would forget to set this prop.

Reply:
@tjuanitas That's what's happening upstream. 

`const isHidden = (size === VIEW_SIZE_TYPE.small || size === VIEW_SIZE_TYPE.medium) && !isLeftNavForcedOpen;`

Reply:
This is possible with event listeners and lifecycle methods. But that would be out of the scope of this ticket